clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -105,6 +105,10 @@ public class CommonClientConfigs {
     public static final int RETRY_BACKOFF_EXP_BASE = 2;
     public static final double RETRY_BACKOFF_JITTER = 0.2;
 
+    public static final String ENABLE_METRICS_PUSH_CONFIG = "enable.metrics.push";
+    public static final String ENABLE_METRICS_PUSH_DOC = "Kafka client telemetry provides Kafka operators improved visibility over the behavior and internals of the clients that use the cluster." +
```

Comment:
The KIP says "Whether to enable pushing of client metrics to the cluster, if the cluster has a client metrics subscription which matches this client." which sounds like a better description to me.

Reply:
Changed the text. Thanks :)